src/SFA.DAS.AssessorService.Application/Interfaces/IStaffCertificateRepository.cs

```diff
@@ -19,14 +21,17 @@ public interface IStaffCertificateRepository
 
     public class StaffReposBatchSearchResult
```

Comment:
why do these classes live in this Repository interface file?

Reply:
There were always there - they should be moved to the DTO area in the domain library